pdns/ws-auth.cc

```diff
@@ -1640,6 +1640,10 @@ static void apiServerAutoprimaries(HttpRequest* req, HttpResponse* resp) {
     if (document["account"].is_string()) {
       master.account = document["account"].string_value();
     }
+
+    if (master.ip=="" or master.nameserver=="") {
+      throw HttpBadRequestException("ip and nameserver fields must be filled");
```

Comment:
i think most of the rest of the api uses ApiException for these cases (= http 422)

Reply:
fixed